### ISSUE TITLE
Add checks that regions are in cost multipliers and load growth maps

### ISSUE THREAD

**Issue by gschivley** [PowerGenome/PowerGenome#28]
Cost multiplier mapping (`cost_multiplier_region_map`) should have all user defined regions and load region mapping (`load_region_map`) should have all base IPM regions.

**Comment by amadabhu** [comment]
I can work on this, I just need to know which file are these functions in? In load_data.py, I could find load_ipm_plant_region_map but I'm not sure if it's related or similar to the load_region_map function here. 
Thanks!

**Comment by gschivley** [comment]
This is mostly about checking the settings file. I started a check settings function [here](https://github.com/PowerGenome/PowerGenome/blob/91d41b75cb69e8ccc6f1a80b88459dc6a589f380/powergenome/util.py#L33-L85). So far it checks to make sure the regions listed in `region_aggregations` are actual IPM regions, that the model regions are in `cost_multiplier_region_map`, and that model regions are in `aeo_fuel_region_map`. I didn't check to see if IPM regions are all appropriately in `cost_multiplier_region_map` because I filled them all in for the example settings file, but it wouldn't be a bad idea to check those too.

If you want to add this check, it would be making sure that all IPM regions that are either a model region or in a region aggregation are listed in one of the lists under `cost_multiplier_region_map`.

There are also some other checks that should be added, like making sure that any user-defined technologies are listed in `cost_multiplier_technology_map`.
